spec/lib/efile/nj/nj1040_calculator_spec.rb

```diff
@@ -904,6 +912,7 @@ def over_65_birth_year
     }
     it 'returns line 39 times tax rate' do
       allow(instance).to receive(:calculate_line_39).and_return 36_000
+      instance.calculate
```

Comment:
[dust] wondering how these were passing without the `instance.calculate`?

Reply:
I think that because they were calling the calculate functions, the mock was able to take effect there, but now it needs the calculate method so that if it's referencing that line, that line value is actually up to date to reflect the mock

Reply:
ah ok makes sense